tests/test_competition.py

```diff
@@ -11,17 +10,3 @@ def test_competition_from_json(test_competition, tmpdir):
     path = test_competition.to_json(str(tmpdir))
     new_competition = CompetitionSpecification.from_json(path)
     assert new_competition == test_competition
-
-def test_competition_evaluation(test_competition):
```

Comment:
This is redundant now since the evaluation logic is the exact same for regular benchmarks.